src/GoogleLogin.js

```diff
@@ -1,55 +1,56 @@
 import React from 'react';
 
-export default class GoogleLogin extends React.Component{
-  constructor(props) {
-    super(props);
-  }
+let auth2;
 
-  componentDidMount () {
-    (function(d, s, id){
-     var js, gs = d.getElementsByTagName(s)[0];
-     if (d.getElementById(id)) {return;}
-     js = d.createElement(s); js.id = id;
-     js.src = 'https://apis.google.com/js/platform.js'
-     gs.parentNode.insertBefore(js, gs);
-   }(document, 'script', 'google-platform'));
-  }
+export default class GoogleLogin extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        let socialId = this.props.socialId, scope = this.props.scope;
+        window.googlePlatformReadyCb = () => {
```

Comment:
this sounds polluting the global scope and looks a bit dirty to me... although couldn't think out of a better way...